libraries/engage/product/selectors/__tests__/variants.spec.js

```diff
@@ -0,0 +1,101 @@
+import { getProductVariants } from '@shopgate/pwa-common-commerce/product';
+import {
+  makeGetProductByCharacteristic,
+  makeGetCharacteristicFeaturedImage,
+} from '../variants';
+import { wrapMemoizedSelector } from '../helpers';
```

Comment:
Sorry... the `wrapMemoizedSelector` module was also added to a PR which is already merged. So i move my comment from the last review here.

Could you please add a more detailed description to this function? It seems to test basic functionality of reselect, since it's an expected behaviour, that a selector returns the same value for multiple calls with the same args and state.

I think, there is no way to run into the error situation, even if the function would be called with a common selector, which was created without the make pattern. The only way to test the selector factories is to call them twice and compare the returned selector instances.